packages/vue/src/types/index.ts

```diff
@@ -1,3 +1,5 @@
+import { ComputedRef, InjectionKey, Ref, Slot } from 'vue';
+import { Interpreter } from 'xstate';
 import {
```

Comment:
Actually screw it, let's do it here.
```suggestion
import { ComputedRef, InjectionKey, Ref, Slot } from 'vue';

import { Interpreter } from 'xstate';

import {
```